.eslintrc.js

```diff
@@ -0,0 +1,239 @@
+/*
+👋 Hi! This file was autogenerated by tslint-to-eslint-config.
+https://github.com/typescript-eslint/tslint-to-eslint-config
+
+It represents the closest reasonable ESLint configuration to this
+project's original TSLint configuration.
+
+We recommend eventually switching this configuration to extend from
+the recommended rulesets in typescript-eslint. 
+https://github.com/typescript-eslint/tslint-to-eslint-config/blob/master/docs/FAQs.md
+
+Happy linting! 💖
+*/
+module.exports = {
+  env: {
+    browser: true,
+  },
+  extends: [
+    'plugin:@typescript-eslint/recommended',
+    'plugin:@typescript-eslint/recommended-requiring-type-checking',
+    'prettier',
+  ],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    project: 'tsconfig.json',
+    sourceType: 'module',
+  },
+  plugins: [
```

Comment:
Maybe you install all these plugins as devDependencies? `npx eslint .` works for me after the install.
<img width="442" alt="Screenshot 2022-09-27 at 13 01 34" src="https://user-images.githubusercontent.com/31354481/192497004-e021c2a6-ba32-48f5-91cf-a244c17b9b5e.png">